### PR TITLE
Disassemble RTC setup dialog and slot machine minigame (bank $24)

### PR DESCRIPTION
**Pull request by ZoomTen**
Also initial cry header pointer and SFX pointer ripping, since I forgot I have those in `master` as well

### CODE REVIEW

File: audio/cries.asm [hunk 1]
```diff
@@ -1,5 +1,46 @@
 INCLUDE "constants.asm"
 
-SECTION "audio/cries.asm", ROMX
+SECTION "audio/cries.asm@Cry Header Pointers", ROMX
+CryHeaderPointers::
+	dbw $3C, $57C5
```
Comment (Rangi42) [inline audio/cries.asm:5]: Can you add shims for each cry header so these can use `dba Cry_Nidoran_M`, etc?

File: audio/sfx.asm [hunk 2]
```diff
@@ -0,0 +1,112 @@
+INCLUDE "constants.asm"
+
+SECTION "audio/sfx.asm@SFX Pointers", ROMX
+SFXPointers::
+	dbw $3C, $43BE
```
Comment (Rangi42) [inline audio/sfx.asm:5]: Likewise add shims for these, even just `Sfx_01`, `Sfx_02`, etc.

File: engine/menu/set_clock_dialog.asm [hunk 3]
```diff
@@ -0,0 +1,349 @@
+INCLUDE "constants.asm"
+
+SECTION "engine/menu/set_clock_dialog.asm", ROMX
+
+UP_ARROW	equ $f0
```
Comment (Rangi42) [inline engine/menu/set_clock_dialog.asm:5]: Capitalize `EQU`.

File: engine/menu/set_clock_dialog.asm [hunk 4]
```diff
@@ -0,0 +1,349 @@
...
+	ret
+
+SetClockDialog_PrintUpDownArrows:
+	ld hl, wClockDialogArrowBlinkCounter
+	ld a, [hl]
+	inc [hl]
+	and 8
+	jr nz, .clear_arrows
+
+;Print up arrow
```
Comment (Rangi42) [inline engine/menu/set_clock_dialog.asm:78]: Space after semicolon: `; Print up arrow`.

File: macros/wram.asm [hunk 5]
```diff
@@ -250,13 +250,13 @@ slot_reel: MACRO
...
-\1Slot0d::       ds 1
-\1Slot0e::       ds 1
-\1Slot0f::       ds 1
+\1ManipCounter:: db
+\1ManipDelay::   db
+\1Field0b::      db
+\1Field0c::      db
+\1Field0d::      db
+\1Field0e::      db
+\1StopDelay::      db
```
Comment (Rangi42) [inline macros/wram.asm:259]: Spacing

File: ram/wram.asm [hunk 6]
```diff
@@ -153,7 +206,7 @@ wGlobalAnimXOffset::
 wc4c8:: db
 
 	ds 7
-
+
```
Comment (Rangi42) [inline ram/wram.asm:209]: Remove trailing space.

**Review by Rangi42 — CHANGES_REQUESTED** [review]
Please see the comments for requested changes.

Also, the new PNGs look unusually large. Check whether they're encoded as 2-bit grayscale. Programs like pngcrush or optipng may fix them; the output of pokecrystal's gfx.py tool is also correctly encoded. (I usually convert a PNG to 2BPP with gfx.py and then back to make sure it's right.)

File: audio/sfx.asm [hunk 7]
```diff
@@ -0,0 +1,112 @@
...
+	dba Sfx_63
+	dba Sfx_64
+	dba Sfx_65
+	dba Sfx_66
+	dba Sfx_67
+	dba Sfx_68
+	dba Sfx_69
+	dba Sfx_6a
+	dba Sfx_6b
+	dba Sfx_6c
```
Comment (Rangi42) [inline audio/sfx.asm:112]: Add newlines at the end of files.

**Review by Rangi42 — CHANGES_REQUESTED** [review]
Please fix that missing newline and the two conflicting files.

**Comment by ZoomTen** [comment]
Just woke up, everything should be fixed hopefully

**Pull request merged**